index.js

```diff
@@ -14,15 +16,21 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
   }
   startIndex = startIndex || 0;
   var string = value[0];
+  var altString;
+  if (platform === 'win32' && typeof string === 'string') {
```

Comment:
as an alternative you could use `path.sep === '\\'`, since that's the only time this normalization needs to happen